### PR TITLE
RPC: Get access keys for account

### PR DESCRIPTION
**Pull request by tifrel**

### CODE REVIEW

**Review by sainthiago — APPROVED** [review]

File: packages/rpc/src/methods/keys.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    allowance: string;
+    receiver_id: string;
+    method_names: string[];
+  };
+}
+
+export const getAccessKeys = async (accountId: string): Promise<AccessKey[]> => {
+  const res = await requestFromNearRpc({
+    jsonrpc: '2.0',
+    id: 'dontcare',
```
Comment (bh2smith) [inline packages/rpc/src/methods/keys.ts:19]: I suppose this is a meaningless field and we can send whatever we want.

File: packages/rpc/src/methods/keys.test.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import fetch from 'isomorphic-unfetch';
+import { getAccessKeys } from './keys';
+
+jest.mock('isomorphic-unfetch');
+
+describe('keys', () => {
```
Comment (bh2smith) [inline packages/rpc/src/methods/keys.test.ts:6]: Could we also add tests that the errors are thrown when the response fields don't exist? That way we would know if someone accidentally deletes one of those "throws".

**Review by bh2smith — APPROVED** [review]
LGTM! Although I think those two extra tests could help out in the future.

File: packages/rpc/src/methods/keys.ts (hunk 1 above)
Comment (tifrel) [reply packages/rpc/src/methods/keys.ts:19]: I just checked the source code and it seems they simply copy it over to the response, so I guess you could use it to order things if you have an async system running, other folks might come up with more fancy use cases ;)

File: packages/rpc/src/methods/keys.test.ts (hunk 2 above)
Comment (tifrel) [reply packages/rpc/src/methods/keys.test.ts:6]: Yup, done

**Pull request merged**